packages/dcos-metrics/build

```diff
@@ -35,7 +35,7 @@ mkdir -p /pkg/src/github.com/dcos
 mv /pkg/src/dcos-metrics /pkg/src/github.com/dcos/dcos-metrics
 
 pushd /pkg/src/github.com/dcos/dcos-metrics
-make build
+make rawbuild
```

Comment:
Why do we need to change the make target?

Reply:
[This change to the Makefile](https://github.com/dcos/dcos-metrics/commit/c7327ad31961e4845ffcfac119352928f1321304) was in master but hadn't been pulled into the 1.11.x branch - there's an issue building via docker which this new make target avoids (cc @philipnrmn for more detail/background on this)

Reply:
Gotcha, I didn't realize this was already on master. 👍